### PR TITLE
feat: Added white borders to Stacked Avatars Component

### LINKED ISSUE #286: style: Add the white border to the StackedAvatars component
image

Code lives here:
1. https://github.com/open-sauced/hot/blob/main/src/components/HotRepositories.tsx#L126-L132
2. https://github.com/open-sauced/hot/blob/main/src/components/PostList.tsx#L69-L77
3. https://github.com/open-sauced/hot/blob/main/src/components/Hero.tsx#L108 - not implemented yet https://github.com/open-sauced/hot/issues/265

### PR DESCRIPTION
**Pull request by yhoungdev**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [X] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This Pull Request fix #286 issue, In this Pull Request I added white borders to stacked avatars component

## Related Tickets & Documents

fix #286
## Mobile & Desktop Screenshots/Recordings

Group 237689

## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [X] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

🤗🤗🎉🎉

### CODE REVIEW

**Comment by yhoungdev** [comment]
Group 237689

**Comment by yhoungdev** [comment]
This Pull Request closes #286

**Comment by bdougie** [comment]
> This Pull Request closes https://github.com/open-sauced/hot/issues/286
Thanks for opening. Note, I update the description in respect to the template.

File: src/components/StackedAvatar.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const StackedAvatar = ({ contributors }: StackedAvatarProps) => (
     {contributors && contributors.slice(0, 5).map(({ contributor, last_merged_at }) => (
```
Comment (bdougie) [inline src/components/StackedAvatar.tsx:8]:
```suggestion
    {contributors?.slice(0, 5).map(({ contributor, last_merged_at }) => (
```
This is complaining, but relevant. Can we test this locally?

Comment (yhoungdev) [reply src/components/StackedAvatar.tsx:8]: yea sure its working perfectly before i pushed

Comment (yhoungdev) [reply src/components/StackedAvatar.tsx:8]: Yea sure working perfectly

File: src/components/StackedAvatar.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ const StackedAvatar = ({ contributors }: StackedAvatarProps) => (
...
       <div
         key={`contributor-avatar-${contributor}`}
-        className="w-6 h-6 overflow-hidden rounded-full transition-all duration-300"
+        className="w-7 h-7 overflow-hidden rounded-full transition-all duration-300 border-[3px] border-white"
```
Comment (bdougie) [inline src/components/StackedAvatar.tsx:11]:
```suggestion
        className="w-7 h-7 overflow-hidden rounded-full transition-all duration-300 border-2 border-white"
```

Forcing this to use tailwind defaults

Comment (bdougie) [inline src/components/StackedAvatar.tsx:11]:
```suggestion
        className="w-7 h-7 overflow-hidden rounded-full transition-all duration-300 border-2 border-white"
```

updating to tailwind default.

**Pull request merged**